apps/team-awesome-web/stories/components/Button.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+import { Button } from "../../src/components/button";
+import { ReactComponent as PlusIcon } from "../../public/icons/plus-icon.svg";
+
+export const Primary = ({ ...args }) => (
+  <Button
+    disabled={args.disabled}
+    onClick={args.onClick}
+    variant={!args.disabled ? "primary" : args.disabled}
+  >
+    {args.icon === "Plus" ? <PlusIcon /> : null}
+    {!args.label ? "Primary button" : args.label}
+  </Button>
+);
+
+export const Secondary = ({ ...args }) => (
+  <Button
+    disabled={args.disabled}
+    onClick={args.onClick}
+    variant={!args.variant ? "secondary" : args.variant}
+  >
+    {args.icon === "Plus" ? <PlusIcon /> : null}
```

Comment:
You can write this also like this `{args.icon === "Plus" && <PlusIcon />}`

Reply:
Ahh makes more sense, it was a leftover from when I nested ternaries for multiple icon selections.